### DISCUSSION TITLE
"reset all proxyman certificates" greyed out

### DISCUSSION THREAD

**Discussion by slayer81** [ProxymanApp/Proxyman#2132] [Q&A] [answered]
Recently, PM for macOS stopped working for me, so I uninstalled it and deleted all the config files. Today, when I reinstalled it, it gave me the error attached, fair enough. 
![iMac Screenshot  2024-09-08 at 12 11 29](https://github.com/user-attachments/assets/fedf15f7-ee98-49f1-a3f7-dd4544a85e76)

But, when I went to reset the certificate, that option is greyed out. (also attached).
![iMac Screenshot  2024-09-08 at 12 13 05](https://github.com/user-attachments/assets/7ac48ed2-a5a9-4846-ad6b-b18fa396f78f)

Can anyone point me in the right direction as to how to remedy this please?

**Comment by NghiaTranUIT** [comment, marked as answer]
It's disabled because there is no certificate that is installed on your Macbook.

You can install it in the Certificate menu -> Install Certificate on this Mac… -> Follow the instructions and make sure it has the green check ✅ 

<img width="812" alt="Screenshot 2024-09-08 at 19 42 59" src="https://github.com/user-attachments/assets/3e5f79cb-1f20-45c9-af0f-5696e157e273">
